src/org/zaproxy/zap/extension/ascan/OptionsScannerPanel.java

```diff
@@ -101,51 +103,62 @@ private JPanel getPanelScanner() {
                     LayoutHelper.getGBC(2, 2, 1, 1.0D, 0, GridBagConstraints.HORIZONTAL, new Insets(2, 2, 2, 2)));
             panelScanner.add(getSliderThreadsPerHost(),
                     LayoutHelper.getGBC(0, 3, 3, 1.0D, 0, GridBagConstraints.HORIZONTAL, new Insets(2, 2, 2, 2)));
-            
+
             panelScanner.add(new JLabel(Constant.messages.getString("ascan.options.maxRes.label")),
                     LayoutHelper.getGBC(0, 4, 1, 0.0D, 0, GridBagConstraints.HORIZONTAL, new Insets(2, 2, 2, 2)));
             panelScanner.add(this.getSpinnerMaxResultsList(),
                     LayoutHelper.getGBC(1, 4, 2, 0.0D, 0, GridBagConstraints.HORIZONTAL, new Insets(2, 2, 2, 2)));
 
+            panelScanner.add(new JLabel(Constant.messages.getString("ascan.options.maxRule.label")),
+                    LayoutHelper.getGBC(0, 5, 1, 0.0D, 0, GridBagConstraints.HORIZONTAL, new Insets(2, 2, 2, 2)));

```

Comment:
The row could be a variable, it would not require changing all statements if a new option needs to be introduced.